cdap-security/src/main/java/co/cask/cdap/security/authorization/AuthorizerInstantiatorService.java

```diff
@@ -99,6 +102,13 @@ public AuthorizerInstantiatorService(CConfiguration cConf, AuthorizationContextF
   @Override
   protected void startUp() throws Exception {
     if (!authorizationEnabled) {
+      LOG.debug("Authorization is disabled. Using a no-op authorizer.");
+      this.authorizer = new NoOpAuthorizer();
+      return;
+    }
+    if (!authenticationEnabled) {
+      LOG.debug("Authorization is enabled. However, authentication is disabled. So all authorization operations will " +
```

Comment:
make the log messages consistent.
`Using a no-op authorizer.`
`... all authorization operations will be no-op.`